CONTRIBUTING.md

```diff
@@ -60,25 +62,34 @@ the maintenance team upon request.
 Please take the following best practices into account if possible. This will reduce
 the work load of the maintainers and to increase the chance that your pull
 request is accepted.
- - Follow [PEP 8](https://peps.python.org/pep-0008/) as a minimal Style Guide
-   for Python Code
- - For docstrings follow [Google Style Guide](https://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html) 
-   (see our own [HOWTO about doc generation](common/doc-dev/1_doc_maintenance_howto.md)).
- - Be careful when using automatic formatters like `black` and please mention
-   the use of it when opening a pull request.
- - Run unit tests before you open a Pull Request. You can run them via
-   `make`-system with `cd common && ./configure && make && make test` or you
-   can use `pytest`. See section [Build and install via `make` system](#build-and-install-via-make-system-recommended) for further details.
- - Try to create new unit tests if appropriated. Use Pythons regular `unittest`
-   instead of `pytest`. If you know the difference please try follow the
-   _Classical (aka Detroit) school_ instead of _London (aka mockist) school_.
-
-# Resources
-
- - [Mailing list _bit-dev_](https://mail.python.org/mailman3/lists/bit-dev.python.org/) for development related topics
- - [Source code documentation for developers](https://backintime-dev.readthedocs.org)
- - [Translations](https://translate.codeberg.org/engage/backintime) are done on a separate platform
- - [HowTo's and maintenance documents](common/doc-dev/README.md)
+
+- Follow [PEP 8](https://peps.python.org/pep-0008/) as a minimal Style Guide
+  for Python Code.
+- For docstrings follow [Google Style Guide](https://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html) 
+  (see our own [HOWTO about doc generation](common/doc-dev/1_doc_maintenance_howto.md)).
+- Be careful when using automatic formatters like `black` and please mention
+  the use of it when opening a pull request.
+- Run unit tests before you open a Pull Request. You can run them via
+  `make`-system with `cd common && ./configure && make && make test` or you
+  can use `pytest`. See section
+  [Build and install via `make`
+  system](#build-and-install-via-make-system-recommended)
+  for further details.
+- Try to create new unit tests if appropriated. Use Pythons regular `unittest`
```

Comment:
No. Line 78 is about which type of unittest to create: regular "unittest" or the fancy pytest-like unittests. I have strong reservations against pytest-style unittests. But line74 is just about how to run the test suite. I do use pytest command as a runner only because the output is fancy and colorful and it can run regular unittests, too.

Reply:
OK I understand. Perhaps you can add this reason to line 74 (because the `pytest` runner has nicer output)...